requirements.in

```diff
@@ -1,10 +1,9 @@
 # dev/testing requirements
 black==18.9b0
-coverage==4.5.4
-pydocstyle==4.0.1
-pylava==0.2.2
-pylava-pylint==0.0.3
-pylint==2.4.4
-pytest==5.3.1
-pytest-asyncio==0.10.0
-pytest-cov==2.8.1
+coverage
+pydocstyle
+pylava
+pylava-pylint
+pylint
+pytest
+pytest-cov
```

Comment:
I'm not this project's author but this change seems unwise. `requirements.txt` is there to have a reproducible dev environment, so version pinning is key. Use setup `install_requires` for providing dependency version ranges  (with proper CI testing coverage for hassle-free/automated version bumps).

I'd also consider this to be outside of the scope of the PR (not atomic).

Reply:
This is not a requirements.txt file, that's still there and everything is pinned. 

I agree it would be better in a different PR. There were dependency conflicts and this was the fastest way to resolve them.

Reply:
I have moved these changes outside of this PR. much cleaner now.